matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/notification/RoomNotificationState.kt

```diff
@@ -25,11 +25,6 @@ enum class RoomNotificationState {
      */
     ALL_MESSAGES_NOISY,
 
-    /**
-     * All the messages will trigger a notification
-     */
-    ALL_MESSAGES,
-
```

Comment:
Sorry, we misunderstood each other.
You should not change this enum defined at the sdk level

I was just asking to remove the Action (that you removed correctly [here](https://github.com/tchapgouv/tchap-android-v2/pull/96/files#diff-3ca374c6ae1d23df09d61e8fe987b35844d6a5ff42cb622799615fb7aa0c1a2fL35)